magento/magento2
# 29217: Fixed -  Prefix and Suffix Translation

### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes magento/magento2#29217  Prefix and Suffix Translation

### Manual testing scenarios (*)


1.From admin panel, Set below option from  Stores -> Configuration -> Customers -> Customer Configuration -> Name and Address Options.
	a. Show Prefix = Yes
	b. Prefix Dropdown Options = Ms.;Mr.;Mrs. (Use Semicolon (;) to separate values)
	c. Show Suffix = Yes
	d. Suffix Dropdown Options = Jr.;Sr.;MD.(Use Semicolon (;) to separate values)
2. Create another store with a different language.
For Example: Create a store german and add a german language pack for this.
Add translation words in german locale file(.csv) that translate "Mr." to "Herr" and "Ms." to "Frau" words and so on.
3. Now check below pages in frontend with different stores(Default store - English Language and German store - German language)
	a.Customer "Create an Account" page.
	b.Login as a customer and open the "My Account" page and Edit Contact Information.
	c.Add any product to the cart as a guest customer and open the checkout page.
 
Before Fix Create Account page

![before_registration](https://user-images.githubusercontent.com/16253886/95655688-b055ae00-0b26-11eb-96de-55dad16b4b96.png)


After Fix Create Account page

![after-registration-page](https://user-images.githubusercontent.com/16253886/95655551-e6def900-0b25-11eb-8f35-a12766638782.png)


Before Fix Checkout Page

![before_checkout](https://user-images.githubusercontent.com/16253886/95655557-ef373400-0b25-11eb-856a-0240a7ded80d.png)

After Fix Checkout Page

![after-checkout_page](https://user-images.githubusercontent.com/16253886/95655564-f65e4200-0b25-11eb-9ed7-7a859d26c7ee.png)


Before Fix Account Details Edit Page

![before_account-edit-page](https://user-images.githubusercontent.com/16253886/95655669-94520c80-0b26-11eb-85ca-a7ff813616c8.png)

After Fix Account Details Edit Page
![after-address-edit-page](https://user-images.githubusercontent.com/16253886/95655577-0d9d2f80-0b26-11eb-97a7-56548d3d9c34.png)



### Questions or comments


### Contribution checklist (*)
